app/indexers/file_set_indexer.rb

```diff
@@ -2,4 +2,54 @@
 
 class FileSetIndexer < Hyrax::FileSetIndexer
   self.thumbnail_path_service = ::ThumbnailPathService
+
+  def generate_solr_document
```

Comment:
I think hyrax does have solr document generate for FileSets but unfortunately it won't support ingest/store multiples files in one fileset at this time. And there are no strategic to store technical metadata for other files other than the original file in the same fileset. So we need to extend it to include technical metadata for all other files in Solr document so that we can expose them for UI displaying.